src/components/Dialog.tsx

```diff
@@ -32,10 +33,10 @@ export const EditDialog = (props) => {
         style={{ width: '350px', margin: '25px', textAlign: 'center' }}
         className={'k-textbox'}
         onChange={handleInputChange}
-        />
+      />
       <DialogActionsBar>
-        <button value={'rename'} className='k-button k-state-selected' onClick={handleDialogClick}>Rename</button>
-        <button value={'cancel'} className='k-button' onClick={handleDialogClick}>Cancel</button>
+        <Button value={'rename'} className='k-button k-selected' onClick={handleDialogClick}>Rename</Button>
```

Comment:
Once we use our Button component we can rethink which classNames we use, too.

Reply:
Right, I will remove both `k-button` and `k-selected` classNames and use themeColor="primary" instead

Reply:
fe579ca